### ISSUE TITLE
Remove a11y strings marked with visible: false from string files

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/tasks#984]
From https://github.com/phetsims/john-travoltage/issues/325, there are 15 strings thorughout the project in translatable strings files marked as `visible: false`. We initially considered this as a strategy to remove a11y strings from the translation utility and mark them for a11y. But we have since moved to using files like *A11yStrings.js to hold a11y strings.

These strings are only used in the PDOM, never displayed visibly. The reason these strings are not translatable yet is because A11y needs the flexibility to change these until we are more confident in them and have had more user feedback.

Any a11y strings marked with `visible: false` should either be removed entirely because they are not used any more or moved to one of the *A11yStrings files.

**Comment by jessegreenberg** [comment]
I should also mention, in https://github.com/phetsims/john-travoltage/issues/325 @jbphet said
>  I think they would need to be examined on a case-by-case basis and, as long as they are not used in the visual portion of the sim, it should be safe to remove them from the strings JSON files.

So this is the inspection I am doing as I go through these.

**Comment by jessegreenberg** [comment]
Done with the above commits. This mostly involved forces-and-motion-basics, but there were 2 strings each in joist and scenery-phet. Most were just removed, as equivalent strings were already in and being used from JoistA11yStrings.js and SceneryPhetA11yStrings.js. The only exception was the accessible label for the Reset All button, which was moved to SceneryPhetA11yStrings.

Closing.
